embed_grader/serapis/static/serapis/custom.css

```diff
@@ -9,3 +9,10 @@
 a:hover {
   cursor: pointer;
 }
+
+.btn-nesl {
+  font-size:12px;
```

Comment:
- Change the class name to `btn-detail-enabled`, I think it better conveys the intention. 
- Put a space after the colons (e.g., `font-size: 12px;`)

I think there's another version which is in grey color, we don't need to address it for now, but we can create a second class name called `btn-detail-disabled`

Reply:
addressed